clang/lib/CConv/ConstraintResolver.cpp

```diff
@@ -516,16 +526,11 @@ void ConstraintResolver::constrainLocalAssign(Stmt *TSt, Expr *LHS, Expr *RHS,
   bool IsAssigned = false;
   std::set<ConstraintVariable *> R =
       getExprConstraintVars(L, RHS, TmpValueCons, LHS->getType(), IsAssigned);
-  // If this is not assigned? Get RVale Cons
+  // If this is not assigned? Get RValCons
   if (!IsAssigned) {
     if (R.empty()) {
       R = TmpValueCons;
     }
-    if (L.empty() && !R.empty()) {
-      std::string Rsn = "Assigning to external expressions.";
-      for (auto CR : R)
-        CR->constrainToWild(Info.getConstraints(), Rsn, &PL);
-    }
```

Comment:
This will break `vsftpd`. This was band-aid for: https://github.com/plum-umd/checkedc-clang/issues/84, until someone fixes https://github.com/plum-umd/checkedc-clang/issues/84, `vsftpd` will break.

Reply:
Understood. We are actively trying to fix #84 so I'm going to go ahead.